overlay.d/20platform-chrony/usr/lib/systemd/system/coreos-platform-chrony-config.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=Call coreos-chrony script that inspects
+# the platform and reconfigures the chrony
```

This looks weird because the statement is broken up across two lines one of them being a comment

Let's just simplify it and get it down to one line: 

```suggestion
Description=Configure chrony based on the platform
```